app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
+		type: Sequelize.STRING,
+		allowNull: false
+	},
+	dispatchedAt: {
+		type: Sequelize.DATE,
+		allowNull: false,
+		field: 'dispatched_at'
+	},
+};
```

Comment:
For the event it looks like there isn't information about which users the event was sent to. This probably shouldn't be sent as part of the payload in all cases, since there may be events that get sent to some users secretly.

Another note is we may want to be able to select all events that were sent to a specific player (for instance if we want to be able to "replay" the game on a disconnect) which means we would want to be able to query in terms of the users who got the message.